ibm/service/cis/data_source_ibm_cis_alert_webhooks.go

```diff
@@ -0,0 +1,90 @@
+// Copyright IBM Corp. 2017, 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package cis
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+const cisWebhooksList = "cis_webhooks_list"
```

Comment:
just make it `cis_webhooks`. change this reference everywhere accordingly.. even in docs